packages/nimble-components/src/table/types.ts

```diff
@@ -25,6 +25,18 @@ export interface TableRecord {
     [key: TableFieldName]: TableFieldValue;
 }
 
+export type StringField<FieldName extends string> = {
```

Comment:
Should have the Table- prefix just because all types are prefixed with associated component.